README.md

```diff
@@ -21,7 +21,7 @@ main
 
```

Comment:
updated the API to match our new  high-level + advanced split.
The examples are much simpler now.
I moved the advanced APIs to `README_ADVANCED.md` to keep this landing page simpler/ less confusing. There's a link out to it.

NOTE : considered pulling tombstone into "high level" space, but then realised we should agree on the API of that before doing that (would be good for it to align with the other high level `findOrCreate`) = seperate PR